### PR TITLE
chore: add entries for knative/func

### PR DESCRIPTION
**Pull request by lance**
This commit moves knative-sandbox/kn-plugin-func and the team members over to knative/func.

See: https://github.com/knative/community/issues/1169

Signed-off-by: Lance Ball <lball@redhat.com>

### CODE REVIEW

File: peribolos/knative-sandbox-OWNERS_ALIASES [hunk 1]
```diff
@@ -114,16 +114,13 @@ aliases:
   - nainaz
   - salaboy
   - zroubalik
-  func-writers:
+  functastic-writers:
```
Comment (psschwei) [inline peribolos/knative-sandbox-OWNERS_ALIASES:117]: For a lot of the other groups, we use the same reader/write/leads groups across both the knative/sandbox lists (so for example, we have serving-writers in both peribolos files, with the same names in each). Then you can also set up separate groups for sandbox repos if you want to allow someone to approve the sandbox but not the core repo ([example](https://github.com/knative-sandbox/container-freezer/blob/main/OWNERS#L4-L5)). Wonder if that might work here too (just thinking out loud, so feel free to disregard)?

**Comment by salaboy** [comment]
does this needs to stay on hold until the repo is finally moved?

**Comment by lance** [comment]
> does this needs to stay on hold until the repo is finally moved?

Yes I think so, unless we want to do it in two steps - add everything for `knative/func` in this PR, and remove from `knative-sandbox/kn-plugin-func` in a second PR.

**Comment by evankanderson** [comment]
/lgtm

Remove the hold when we're ready to pull the trigger.

**Review by evankanderson — APPROVED** [review]
/hold cancel

**Pull request merged**